calendar-bundle/tests/Security/Voter/CalendarContentVoterTest.php

```diff
@@ -64,8 +64,8 @@ public function testChecksElementAccessPermission(CreateAction|DeleteAction|Read
 
         foreach ($parentRecords as $id => $records) {
             $fetchAllAssociativeMap[] = [
-                'SELECT id, @pid:=pid AS pid, ptable FROM tl_content WHERE id=?'.str_repeat(' UNION SELECT id, @pid:=pid AS pid, ptable FROM tl_content WHERE id=@pid', 9),
-                [$id],
+                'SELECT id, @pid:=pid AS pid, ptable FROM tl_content WHERE id=?'.str_repeat(' UNION SELECT id, @pid:=pid AS pid, ptable FROM tl_content WHERE id=@pid AND ptable=?', 9),
+                [$id, ...array_fill(0, 9, 'tl_content')],
```

Comment:
```suggestion
                'SELECT id, @pid:=pid AS pid, ptable FROM tl_content WHERE id=?'.str_repeat(" UNION SELECT id, @pid:=pid AS pid, ptable FROM tl_content WHERE id=@pid AND ptable='tl_content'", 9),
                [$id],
```

Reply:
That would be wrong, the table name is dynamic

Reply:
True 👍